club/theme.json

```diff
@@ -129,26 +129,88 @@
 					]
 				}
 			],
+            "fluid": true,
             "fontSizes": [
                 {
-                    "name": "Small",
+                    "size": "0.875rem",
+                    "fluid": {
+                        "min": "0.875rem",
+                        "max": "1rem"
+                    },
+                    "slug": "small",
+                    "name": "Small"
+                },
+                {
                     "size": "1rem",
-                    "slug": "small"
+                    "fluid": {
+                        "min": "0.875rem",
+                        "max": "1.25rem"
+                    },
+                    "slug": "default",
+                    "name": "Default"
                 },
                 {
-                    "name": "Medium",
-                    "size": "1.5rem",
-                    "slug": "medium"
+                    "size": "1.25rem",
+                    "fluid": {
+                        "min": "1.25rem",
+                        "max": "1.5rem"
+                    },
+                    "slug": "medium",
+                    "name": "Medium"
                 },
                 {
-                    "name": "Large",
                     "size": "1.75rem",
-                    "slug": "large"
+                    "fluid": false,
+                    "slug": "large",
+                    "name": "Large"
+                },
+                {
+                    "size": "2.5rem",
+                    "fluid": false,
+                    "slug": "x-large",
+                    "name": "Extra Large"
+                },
+                {
+                    "size": "3.75rem",
+                    "fluid": {
+                        "min": "3.75rem",
+                        "max": "8.125rem"
+                    },
+                    "slug": "header-one",
```

Comment:
I'm not sure its a good idea to implement these non-standard typography options. This will expose these to users in the block editor and they will be able to select them for their content. Then when they switch themes they will lose these settings and the fonts will revert to a default size. I'd recommend either moving these to custom, or simply defining them directly on the patterns and templates as needed.

Reply:
I'm also not sure it's a good idea to include the non-standard options. Should we move the non-standard options to where they're being used (e.g. to the heading elements, directly in the templates, etc) for now, until we have a solution like the hidden sizes? Then we can progress this PR.